releasenotes/notes/c-api-version-support-55b018bf9552db1f.yaml

```
@@ -0,0 +1,20 @@
+---
+features_c:
+  - |
+    Added support for querying the Qiskit version information from the C API via the following macros:
+    - `QISKIT_VERSION_MAJOR` : Contains the major version number.
+    - `QISKIT_VERSION_MINOR` : Contains the minor version number.
+    - `QISKIT_VERSION_PATCH` : Contains the patch version number.
+    - `QISKIT_VERSION` : Contains a numeric representation of the version information, which can be used for comparisons.
+    - `QISKIT_VERSION_NUMERIC(M,m,p)` : Function-like Macro that returns the version "M.m.p" as a numeric value, which can be used for comparisons.
+
+    for example, to check if the current version is at least 2.1.0, you can use:
+    ```c
+    if (QISKIT_VERSION >= QISKIT_VERSION_NUMERIC(2, 1, 0)) {
+        // Code for version 2.1.0 or later
+    }
+    ```
```

The reno release notes use sphinx and restructured text. So the markdown formatting you used here won't render correctly:

```suggestion
features_c:
  - |
    Added support for querying the Qiskit version information from the C API via the following macros:
    
    * ``QISKIT_VERSION_MAJOR`` : Contains the major version number.
    * ``QISKIT_VERSION_MINOR`` : Contains the minor version number.
    * ``QISKIT_VERSION_PATCH`` : Contains the patch version number.
    * ``QISKIT_VERSION`` : Contains a numeric representation of the version information, which can be used for comparisons.
    * ``QISKIT_VERSION_NUMERIC(M,m,p)`` : Function-like Macro that returns the version "M.m.p" as a numeric value, which can be used for comparisons.

    For example, to check if the current version is at least 2.1.0, you can use:

    .. code-block: c

        if (QISKIT_VERSION >= QISKIT_VERSION_NUMERIC(2, 1, 0)) {
            // Code for version 2.1.0 or later
        }
```